packages/core/src/cli/add-offical-connectors.ts

```diff
@@ -0,0 +1,16 @@
+import 'module-alias/register';
+import { getEnv } from '@silverhand/essentials';
+
+import { addOfficalConnectors } from '@/connectors/add-connectors';
+import { defaultConnectorDirectory } from '@/env-set';
+
+import { configDotEnv } from '../env-set/dot-env';
+
+configDotEnv();
+
+const addOfficalConnectorsCli = async () => {
+  const connectorDirectory = getEnv('CONNECTOR_DIRECTORY', defaultConnectorDirectory);
```

Comment:
this `connectorDirectory` seems shared by a few places, maybe we can define them instead of getting their value every time we need it?

Reply:
We need to load this by env, is the only way to get it.